### ISSUE TITLE
Disabled switches aren't indicated as such with voiceover

### ISSUE THREAD

**Issue by guidezpl** [flutter/flutter#88046]
Steps to reproduce
1. Run an app with a disabled Material or Cupertino switch on an iOS device
2. Turn on voiceover
3. Notice the disabled switches are announced just like non-disabled switches

By comparison, disabled radio buttons are announced as 'disabled'

**Comment by HansMuller** [comment]
CC @chunhtai

**Comment by chunhtai** [comment]
If for disabled you meant inactive switch button (onchange = null), then it is not focusable by voiceover. How do you make voiceover announce the disabled switches?

It may be easier if you can provide a reproducible example

**Comment by guidezpl** [comment]
I was looking at code that wrapped switches in `Semantics` and didn't do anything special for disabled switches, so WAI
